sql/gen-sql-functions-docs.py

```diff
@@ -106,7 +106,7 @@ def _make_pretty_usage(infos):
     for info in infos:
         # Extracts (signature, description) pairs from `info.usage`, e.g.,
         # the signature is `func(expr)` and the description is `...` in an usage `func(expr) - ...`.
```

Comment:
Do we need to update this description accordingly because this PR dropped `(` and `)` requirement?